readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -154,3 +148,13 @@ class MkdocsJSON(BaseMkdocs):
     type = 'mkdocs_json'
     builder = 'json'
     build_dir = '_build/json'
+
+    def build(self, **kwargs):
```

Comment:
Does this need to be subclassed? It seems like the same logic as the class above?